### PR TITLE
Add node test of logical operators.

### PR DESCRIPTION
**Pull request by guoyuhong**

### CODE REVIEW

File: onnx/backend/test/case/node/and.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        node = onnx.helper.make_node(
+            'And',
+            inputs=['x', 'y'],
+            outputs=['and'],
+        )
+
+        x = (np.random.randn(3, 4, 5) > 0).astype(np.bool)
+        y = (np.random.randn(3, 4, 5) > 0).astype(np.bool)
+        z = np.logical_and(x, y)
+        expect(node, inputs=[x, y], outputs=[z],
```
Comment (linkerzhang) [inline onnx/backend/test/case/node/and.py]: Thank you very much! Would you also help to add cases to cover,
1. broadcast not specified, and two inputs have different shape.
2. cases for different attribute values of "axis".

**Comment by bddppq** [comment]
The data files need to be generated and committed to git as well
@guoyuhong @linkerzhang

**Comment by bddppq** [comment]
ping @linkerzhang and @guoyuhong for adding the missing data files. You can see the command of generating them in this doc: https://github.com/onnx/onnx/blob/master/docs/OnnxBackendTest.md

**Comment by guoyuhong** [comment]
@bddppq Thanks for the reminding. I will try to add this data files. One more question: it looks like all the node tests are skipped. How to guarantee that our unit-test code is correct?

**Comment by bddppq** [comment]
You will need to have a backend to actually run the tests.

**Comment by guoyuhong** [comment]
@bddppq I write this test based on your previous node test examples. I will try to apply this unit test to a real backend to have a try. It may take some time.

**Comment by bddppq** [comment]
@guoyuhong If it could save you sometime: I have tested your new cases with onnx-caffe2 :-)

**Comment by guoyuhong** [comment]
@bddppq. For a freshman, they may not know how to test it in a real back-end. Do we need some documentation on how to run a real back-end test to verify the code?

**Comment by bddppq** [comment]
@guoyuhong Sure that would be helpful. Could you document it while you are exercising this process this time?
And besides, could you upload the generated data files soon? Right now new contributors need to manually filter out them before uploading the corresponding data files for their new test cases.

**Pull request merged**